src/components/logger/index.ts

```diff
@@ -212,3 +217,10 @@ export async function printAllLogs() {
     })
   })
 }
+
+em.on(PROBE_LOGS_BUILT, async () => {
+  // Finally save these logs into database
+  // TODO: add log persistence functions here
+  // await saveProbeRequestLog(mLog)
```

Comment:
this line is intended comment or we forgot to uncomment ya?

Reply:
intended for some future planning... maybe.

Reply:
I think it's better to comment:

```
// TODO: Handle event when probe logs has been built
```

Considering the probe request log and notification log is already being saved on the top.

Reply:
yup better. thanks